src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -1405,29 +1492,112 @@ const cleanupSubscriptions = () => {
 </Block>
 </BlockSwitcher>
 
-[comment]: # (EXPANDER end)
-### Unsubscribe from a subscription
+</Accordion>
+
+<br/>
+
+<Accordion title='Create a custom GraphQL subscription by ID' headingLevel='4' eyebrow='Walkthrough'>
```

Comment:
Added the other guide mentioned in the comments from before for this expander. This one reads more as an example use case so I wonder if we might move it to that page instead? Kept it so we can have the conversation on where it can go so we can begin getting rid of the guides menu items.

Reply:
let's keep it in for now but I think there's opportunity to reduce this into a short blurb and move the main content to "custom business logic"